src/modifiers/containing_scope.py

```diff
@@ -1,80 +1,91 @@
-from talon import Module, app
```

Comment:
Looks like line endings got messed up again

Reply:
This is such a hassle. Let me try an up the git setting and see if it becomes better in the future